### PR TITLE
[IMP] * : add missing borders

### PR DESCRIPTION
**Pull request by Brieuc-brd**
Depending on Milk, this commit adds some missing borders.

Requires: 
- https://github.com/odoo-dev/enterprise/pull/533

task-2818586

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
Hello @stefanorigano  👋  This PR handles the missing borders.
I'll investigate on the others points we discussed in another branch.

File: addons/web/static/src/search/search_panel/search_panel.scss [hunk 1]
```diff
@@ -22,7 +22,11 @@
 
 .o_search_panel {
     width: var(--SearchPanel-width, #{$o-search-panel-width});
+    border-right: $border-width solid $border-color;
     font-size: var(--SearchPanel-fontSize, #{$o-search-panel-font-size});
+    @include media-breakpoint-down(md) {
```
Comment (stefanorigano) [inline addons/web/static/src/search/search_panel/search_panel.scss]: Can we do the other way around? eg:
```scss
@include media-breakpoint-up(md) {
    border-right: $border-width solid $border-color;
}
```

**Review by stefanorigano — CHANGES_REQUESTED** [review]

File: addons/web/static/src/search/search_panel/search_panel.scss (hunk 1 above)
Comment (Brieuc-brd) [reply addons/web/static/src/search/search_panel/search_panel.scss]: Of course ! It's much better ! Changes done 👍 (also for Enterprise)

**Comment by stefanorigano** [comment]
Thanks 👍

**Pull request merged**